sale_order_min_qty/models/product_category.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class ProductCategory(models.Model):
+    _name = "product.category"
+    _inherit = ["product.category", "product.min.multiple.mixin"]
+
+    def _get_sale_min_multiple_qty(self):
+        res = super(ProductCategory, self)._get_sale_min_multiple_qty()
```

Comment:
In python 3 no more super with the same class name than current one
```suggestion
        res = super()._get_sale_min_multiple_qty()
```